src/validation.cairo

```diff
@@ -63,3 +66,45 @@ fn validate_merkle_root(self: @ChainState, block: @Block) -> Result<(), ByteArra
     // TODO: implement
     Result::Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use core::result::ResultTrait;
+    use super::validate_target;
+    use super::{Block, ChainState,};
+    use super::super::state::{Header, Transaction, TxIn, TxOut};
+    use core::array::{Span};
+
+    #[test]
+    fn test_validate_target() {
+        let mut chain_state = ChainState {
+            block_height: 1,
+            total_work: 1,
+            best_block_hash: 1,
+            current_target: 1,
+            epoch_start_time: 1,
+            prev_timestamps: array![1, 2, 3, 4, 5].span(),
+        };
+        let txIn = TxIn { txid: 1, index: 1, script: @"1", sequence: 1, };
+        let txOut = TxOut { value: 1, pk_script: @"1", };
+        let transaction = Transaction {
```

Comment:
There is no need to add transactions to the block in this test.

Reply:
Okay, I removed it.
Does the rest look good? Maybe add some tests?